tailwind.config.js

```diff
@@ -21,6 +21,7 @@ export default {
         'grayscale-10': 'hsl(var(--grayscale-10) / <alpha-value>)',
         'grayscale-20': 'hsl(var(--grayscale-20) / <alpha-value>)',
         'grayscale-30': 'hsl(var(--grayscale-30) / <alpha-value>)',
+        'grayscale-40': 'rgb(219 223 231 / <alpha-value>)',
```

Comment:
I am defining this one differently because having the rgb value here is easier and we don't need the CSS variables.
I am not updating the other variables because we will remove them and replace them with variables with proper names shortly